cob_default_robot_config/cob4-2/command_gui_buttons.yaml

```diff
@@ -71,8 +71,7 @@ group17:
   group_name: torso
   component_name: behavior
   buttons: [[torso_front,trigger,torso_front],
-    [torso_front_home,trigger,torso_front_home],
-    [pick,trigger,pick]]
+    [torso_front_home,trigger,torso_front_home]]
```

Comment:
I'd keep it out for the moment and bring it back as soon as #411 is tested and merged